ontology/docs/hvac_chws.md

```diff
@@ -0,0 +1,329 @@
+# Chilled & Condensing Water Systems (CHWS & CWS)
+
+## Type Definition
+Hydronic cooling and condensing systems. Chilled water systems cool water mechanically, which circulates to end-use devices; they dump the heat absorbed from the chilled water loop either to atmosphere ('air-cooled') or to another water distribution system ('water-cooled'). Condensing systems serve those mechanical devices which rely on water-source heat rejection.
+
+There are many varieties of chilled & condenser water system, but some common examples include:
+- **Constant volume** (predominantly on condensing water systems), where pumps run at constant speeds providing fixed flowrates at given stages.
+- **Variable primary**, where there is one loop between the chillers and the end devices. The pumps operate at variable speed.
+- **Primary/Secondary**, where there are two separate loops, with the primary serving as the production loop and the secondary serving as the distribution loop.
+
+## Type Requirements
+- Chillers require the mechanical cooling of water.
+- Condensers require that, regardless of their own cooling method (sensible or evaporative), they serve downstream devices which require water as a condensing source.
+- Heat exchangers provide heat transfer between two fluid streams, without intermixing the fluids. They are modeled separately only if they are stand-alone devices. 
+
+## Example: Variable Primary CHWS (Air-Cooled) 
+This version of CHWS has a single set of variable speed pumps which serve air-cooled chillers and downstream equipment.
+
+### BMS Example
+![CHWS Variable Primary](./figures/bms_screenshots/chw_aircooled.png)
+**Notes:**
+- The fields associated with individual devices (individual chiller commands, for example) get mapped to those devices; system level telemetry gets assigned to the system directly.
+
+### System Diagram and Connections
+![CHWS Variable Primary](./figures/system_diagrams/chws_aircooled.png)
+
+### Sample Building Config
+```yaml
+BLDG-1:
+  type: FACILITIES/BUILDING
+
+ZONE-1:
+  connections:
+    BLDG-1: CONTAINS
+  type: FACILITIES/ZONE
+
+ZONE-2:
+  connections:
+    BLDG-1: CONTAINS
+  type: FACILITIES/ZONE
+
+CHWSYS-BLDG-1:
+  connections:
+    BLDG-1: CONTAINS
+    FCU-1: FEEDS
+    AHU-1: FEEDS
+  type: HVAC/HWS_...
+  translation:
+    supply_water_temperature_sensor:
+      present_value: points.sys_swt.present_value
+      units:
+        key: pointset.points.sys_swt.units
+        values:
+          degrees_fahrenheit: 'deg-F'
+    differential_pressure_sensor:
+      present_value: points.sys_dp.present_value
+      units:
+        key: pointset.points.sys_dp.units
+        values:
+          pascals: 'Pa'
+    ...
+
+CHL-1:
+  connections:
+    BLDG-1: CONTAINS
+    CHWSYS-BLDG-1: CONTAINS
+  type: HVAC/CHL_...
```

Comment:
type: should be HVAC/CH_...  (remove L)
Same for ACC below

Reply:
fixed